### ISSUE TITLE
invalid URI

### ISSUE THREAD

**Issue by thanhtai9606** [bartvanhoey/AbpAddCustomClaimToAccessToken#1]
"An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set."
I cannot call rest Api from local with your example! Help me please

**Comment by bartvanhoey** [comment]
@thanhtai9606 

the portnumber should be the same number as the API portnumber, I guess ...

**Comment by thanhtai9606** [comment]
> @thanhtai9606
> 
> the portnumber should be the same number as the API portnumber, I guess ...

yes, I already set the same port with Api. But it not working :(. It show the same error

**Comment by thanhtai9606** [comment]
maybe it miss  
`httpClient.Value.BaseAddress = new Uri(authority);`
